nixos/modules/services/logging/syslog-ng.nix

```diff
@@ -85,10 +85,11 @@ in {
       after = [ "multi-user.target" ]; # makes sure hostname etc is set
       serviceConfig = {
         Type = "notify";
+        PIDFile = "${pidFile}";
```

Comment:
Is pidfile required for this to work?

Reply:
Worked without PIDFile = "${pidFile}"; Remove?

Reply:
systemd does some auto-detection for the main pid through forks, but I think since we have the pidfile already, we might as well pass it to be sure.

Is there a problem with `PIDFile = pidFile;`? I'm pretty sure the `"${..}` isn't needed, which is why I proposed this code in the first place.

Reply:
Variant ExecReload = "${pkgs.coreutils}/bin/kill -HUP `${pkgs.coreutils}/bin/cat ${pidFile}`"; fully working, for the first time and made such a.

Reply:
No, use this:

```nix
{
  PIDFile = pidFile;
  ExecReload = "${pkgs.coreutils}/bin/kill -HUP $MAINPID";
}
```

As I said initially